README.md

```diff
@@ -68,3 +68,25 @@ _These instructions only apply, when docker-compose is set up on a server machin
 * Delete Local Docker images and volumes `dc down -v --rmi local`
 * Clear docker system and volume cache  `yes | d system prune && yes | d volume prune`
 * Clear all docker images and containers not in use `yes | d system prune -a`
+
+## Adding a new, real Source Log ##
+To the appropriate config file (such as `gossec.cfg`), copy the format for a source log. Use information from the [List of Known Logs](https://www.gstatic.com/ct/log_list/v2/log_list.json).
```

Comment:
"Use information....." Does this mean that this is where we get the information or as an example?

Reply:
the link is where we get information from. it is the official list of CT loggers

Reply:
(take a look at the page and see if you have more questions)